pages/library/_viewId.vue

```diff
@@ -1,22 +1,22 @@
 <template>
   <div>
     <h1>{{ Name }}</h1>
-    <v-list>
-      <v-list-item
+    <div class="cardsContainer">
+      <v-card
```

Comment:
This should probably be moved to a dedicated component. Something like <item-card>, perhaps.

Reply:
Sounds good, I'll move it to a `cardbuilder` component tomorrow

Reply:
Done